theano/gpuarray/dnn.py

```diff
@@ -1135,6 +1135,11 @@ def dnn_conv3d(img, kerns, border_mode='valid', subsample=(1, 1, 1),
                                     desc_op.subsample)
     out_shp = assert_conv_shape(out_shp)
     out = gpu_alloc_empty(ctx_name, dtype=img.dtype)(*out_shp)
+    if config.dnn.conv.algo_bwd_filter != 'none':
+        warnings.warn('[new backend] cuDNN backward filter convolution computation for 3D convolutions '
```

Comment:
It seems it indeed depends on the subsample (when input shape, filter shape, border mode [=full] and conv mode [='conv'] are constant but subsample changes). But I can't find a subsample pattern that is always associated to an error, so it is currently difficult to tell which kind of subsamples can produce an error with algo 3.